src/main/java/com/lightstep/tracer/shared/SpanBuilder.java

```diff
@@ -47,11 +46,11 @@
     }
 
     @Override
-    public Tracer.SpanBuilder asChildOf(BaseSpan<?> parent) {
+    public Tracer.SpanBuilder asChildOf(io.opentracing.Span parent) {
         if (parent == null) {
             return this;
         }
-        return asChildOf(parent.context());
+        return addReference(CHILD_OF, parent.context());
```

Comment:
Why is `asChildOf` replaced here? Will `asChildOf(parent.context())` still work here?

Reply:
fixed